src/error.rs

```diff
@@ -1,9 +1,6 @@
 //! Error types
 
 use crate::Config;
-use crossbeam_channel;
-#[cfg(target_os = "linux")]
-use mio_extras;
```

Comment:
Was that unused ?

Reply:
Single path importing is unnecessary in the 2018 edition.